### ISSUE TITLE
Schema common type names cannot be built-in type names

### ISSUE THREAD

**Issue by shaobo-he-aws** [cedar-policy/cedar#973]
### Before opening, please confirm:

- [X] I have [searched for duplicate or closed issues](https://github.com/cedar-policy/cedar/issues?q=is%3Aissue+).
- [X] I have read the guide for [submitting bug reports](https://github.com/cedar-policy/cedar/blob/main/CONTRIBUTING.md#bug-reports).
- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

### Bug Category

Schemas and Validation

### Describe the bug

JSON schema `{"N":{"commonTypes":{"Long":{"type":"Long"}},"entityTypes":{},"actions":{}}}` is not considered valid and has an error message "duplicate common type type `Long`". 

### Expected behavior

It should be valid.

### Reproduction steps

echo '{"N":{"commonTypes":{"Long":{"type":"Long"}},"entityTypes":{},"actions":{}}}' | cargo run translate-schema --direction json-to-human

### Code Snippet

```rust
// Put your code below this line.

```

### Log output

```
// Put your output below this line

```

**Comment by shaobo-he-aws** [comment]
Upon further investigation, this behavior is not unexpected because there's currently no way to disambiguate built-in types and common types declared in the empty namespace. For instance,

```JSON
{"":{"commonTypes":{"Long":{"type":"Long"}},"entityTypes":{},"actions":{}}}
```

It's unclear which type is referred when using `Long` in the empty namespace.

**Comment by shaobo-he-aws** [comment]
Theoretically, we can disambiguate these two kinds of types once [RFC 52](https://github.com/cedar-policy/rfcs/pull/52) is implemented (i.e., `__cedar::Long` vs `Long`).

**Comment by shaobo-he-aws** [comment]
Fixed by #974
